client/python/armada_client/client.py

```diff
@@ -89,36 +123,72 @@ def reprioritize_jobs(
         return response
 
     def create_queue(self, name: str, **queue_params):
-        """Implementation of CreateQueue rpc function"""
+        """Create the queue by name.
+
+        TODO(JayF) what are queue params?
```

Comment:
Its a proto object that represents the Queue object.  

Its a nested object.

https://github.com/G-Research/armada/blob/master/pkg/api/submit.proto#L89

Reply:
That's the type in general terms, yes, but it's not the python type of the object. That's what I really need to make these strings more accurate with typing. 

I did mean to remove these TODOs though, I'll do that now.

Reply:
ACK.

Reply:
Fixed via amended commit.